clarity/Clarinet.toml

```diff
@@ -976,3 +976,9 @@ depends_on = [ "trait-sip-010", "trait-ownable" ]
 [contracts.bridge-endpoint]
 path = "contracts/bridge/bridge-endpoint.clar"
 depends_on = ["trait-ownable", "trait-sip-010"]
+
+[contracts.amm-swap-pool]
+path = "contracts/pool/amm-swap-pool.clar"
+
+[contracts.token-amm-swap-pool]
+path = "contracts/pool-token/token-amm-swap-pool.clar"
```

Comment:
@fiftyeightandeight these two contracts have to deps? is that expected?

Reply:
https://github.com/alexgo-io/alex-v1/commit/d4798d367e8f0d0330a33811961d73d957da846e

Reply:
the latest version of clarinet removed this requirement (of `depends_on`). If you need them for our own use, I can certainly add them?

Reply:
Yeah, we need them to determine the order of contract deployments. It'd be nice if we can keep them.

Reply:
Ok, I see you already made changes to the `dev`, so assume all good.